.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+# Python versions to test
+python:
+    - "3.5"
+# Command to install dependencies
+install:
+    - pip install -r requirements.txt
```

Comment:
do we need this step as the requirements are installed as part of setup.py?

Reply:
True, I'll go ahead and remove this line